x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -78,8 +107,13 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
   tableId,
   to,
   filterGroup,
+  runtimeMappings,
+  signalIndexName,
 }) => {
   const dispatch = useDispatch();
+  const [selectedGroup, setSelectedGroup] = useState<string | undefined>(
+    storage.get(`${ALERTS_TABLE_GROUPS_SELECTION_KEY}-${tableId}`)
+  );
```

Comment:
Just noting that there is no space-awareness with this key, so this selection will follow users between spaces. This is the same behavior as other settings we've been storing in localstorage (like Rules table filters and such), so it matches current expectations, but wanted to note since technically it could leak field names between spaces (in the event there are custom alert mappings per space).

I think the bigger issue here is that it doesn't appear as if we're clearing localstorage on a `logout`, so settings could be leaked from one session to another. Or perhaps this is only for dev builds? 🤷‍♂️ 
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/216434653-de574f52-f9ab-495d-be84-d8904692d13e.png" />
</p>

Reply:
Yeah, you are right. I think adding a space awareness will be a good idea. And yes, we should probably follow the user setting per space in other places in Security.